go.mod

```diff
@@ -5,6 +5,7 @@ go 1.22
 toolchain go1.22.8
 
 require (
+	github.com/antithesishq/antithesis-sdk-go v0.4.3-default-no-op
```

Comment:
So anyone who tries to install the server this will also be required?

Reply:
It's a no-op version, so the entire SDK is stubbed out with empty functions which get inlined away unless the `enable_antithesis_sdk` build tag is passed in, which it only is under Antithesis.